crates/dbs-allocator/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "db-allocator"
+name = "dbs-allocator"
```

Comment:
Please help to add `description = ""` to Cargo.toml too.

Reply:
added